app/models/user.rb

```diff
@@ -1,5 +1,12 @@
 class User < ApplicationRecord
-  has_many :posts
-  has_many :comments
-  has_many :likes
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+  after_initialize :set_defaults
+
+  private
+
+  def set_defaults
+    self.postsCounter ||= 0
+  end
 end
```

Comment:
- [ ] The User model should have a method for returning the 3 most recent posts.

  ```
  def recent_posts
    posts.order('created_at Desc').limit(3)
  end
  ```